pkg/remediator/remediator.go

```diff
@@ -108,7 +160,26 @@ func (r *Remediator) Start(ctx context.Context) <-chan struct{} {
 		defer close(doneCh)
 		wg.Wait()
 	}()
-	return doneCh
+
+	r.doneCh = doneCh
+	r.stopFn = cancel
+}
+
+// Pause the Remediator by stopping the workers and waiting for them to be done.
+func (r *Remediator) Pause() {
+	klog.V(1).Info("Remediator pausing...")
```

Comment:
Does this need to be synchronized with lifecycleMux? Seems that way since we are using `stopFn` and `doneCh`

Reply:
I was thinking that the channel and cancel func were thread-safe, but you're right, the variables aren't. We probably don't want Start or Resume to be called while Pause is being called. They won't with the current usage, but I'll add the lock to be defensive.